5    1
6    1
7    1
8    1
9    1
Name: 0, dtype: int64
Training Loop:
Traceback (most recent call last):
  File "/Users/josephinewang/Desktop/INtro to ML/CMPM-17-FinalPro/main.py", line 145, in <module>
    train_loop(train_loader)  # Running the training loop
  File "/Users/josephinewang/Desktop/INtro to ML/CMPM-17-FinalPro/main.py", line 113, in train_loop
    loss = criterion(output, target)  # CrossEntropyLoss expects class labels as integers
  File "/Users/josephinewang/Desktop/INtro to ML/.venv/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1739, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/Users/josephinewang/Desktop/INtro to ML/.venv/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1750, in _call_impl
    return forward_call(*args, **kwargs)
  File "/Users/josephinewang/Desktop/INtro to ML/.venv/lib/python3.10/site-packages/torch/nn/modules/loss.py", line 1295, in forward
    return F.cross_entropy(
  File "/Users/josephinewang/Desktop/INtro to ML/.venv/lib/python3.10/site-packages/torch/nn/functional.py", line 3494, in cross_entropy
    return torch._C._nn.cross_entropy_loss(
RuntimeError: 0D or 1D target tensor expected, multi-target not supported
